ui/signed-out/src/androidMain/kotlin/social/androiddev/signedout/selectserver/redirectScheme.kt

```diff
@@ -9,4 +9,4 @@
  */
 package social.androiddev.signedout.selectserver
 
-actual val redirectScheme: String get() = "oauth2redirect"
+actual val redirectScheme: String get() = "socialandroiddevdodo"
```

Comment:
Please add a comment here for context for the reader, that this _must_ match the scheme set in the intent in the manifest

Reply:
technically the **Dodo** app is not related to androiddev.social, should we use something more generic?

Reply:
Oh, great catch. Yes I agree with this ^

Reply:
Thoughts? It needs to be unique so I was worried just dodo would be too short

Reply:
`dodooauth2redirect`?

Reply:
Changed